client/package.json

```diff
@@ -5,6 +5,10 @@
   "dependencies": {
     "@emotion/react": "^11.6.0",
     "@emotion/styled": "^11.6.0",
+    "@material-ui/core": "^4.12.3",
+    "@material-ui/icons": "^4.11.2",
+    "@material-ui/lab": "^4.0.0-alpha.60",
+    "@material-ui/pickers": "^3.3.10",
```

Comment:
You've installed v4 of Material UI, we're on v5 of Material UI. Please remove these packages.